### PR TITLE
Fix negative firestacks not being able to be obtained

### LINKED ISSUE #63966: You can't get wet
Local testing. 

Negative firestacks (wetness) was a thing. It doesn't work anymore. You can't get firestacks under 0. 

## Reproduction:

1. Splash yourself with water or stand under a shower. 
2. Examine yourself. It says nothing about being wet. 
3. Check firestacks. It doesn't go below 0. 

This breaks the alkali perspiration symptom (tested) which would make you explode if wet.

### PR DESCRIPTION
**Pull request by pitreforten**
## About The Pull Request

Switches around the order of some code to fix wetstacks.

## Why It's Good For The Game

Fixes #63966

## Changelog

:cl:
fix: you can now get wet again!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1366,9 +1366,9 @@
  * * stacks: int The amount to set fire_stacks to
  */
 /mob/living/proc/set_fire_stacks(stacks)
-	fire_stacks = clamp(stacks, -20, 20)
-	if(fire_stacks <= 0)
+	if(stacks <= 0)
 		extinguish_mob()
+	fire_stacks = clamp(stacks, -20, 20)
```
Comment (TemporalOroboros) [inline code/modules/mob/living/living.dm:1371]: This makes it so decreasing your fire stacks below 0 won't automatically extinguish the mob. Can you make sure this doesn't break something?

**Comment by MrMelbert** [comment]
Might be smarter to just change the `fire_stacks = 0` in `extinguish_mob()` to something like `if (firestacks >= 0) firestacks = 0`

**Comment by SuperNovaa41** [comment]
If you change your description to Fixes https://github.com/tgstation/tgstation/issues/63966 or Closes https://github.com/tgstation/tgstation/issues/63966 it will auto close it when this pr gets merged

**Comment by optimumtact** [comment]
Ben Shapiro in tatters

**Pull request merged**